pkg/storaged/dialog.jsx

```diff
@@ -379,6 +380,7 @@ export const dialog_open = (def) => {
     };
 
     function run_action(progress_callback, variant) {
+        clicked_variant = variant;
```

Comment:
I think we can just pass `variant` to `validate` as an argument. No need to hide it in a global variable.

Reply:
Right, updated :grin:

Reply:
Perfect!